packages/docusaurus-init/templates/bootstrap/docusaurus.config.js

```diff
@@ -1,88 +1,93 @@
-/** @type {import('@docusaurus/types').DocusaurusConfig} */
-module.exports = {
-  title: 'My Site',
-  tagline: 'The tagline of my site',
-  url: 'https://your-docusaurus-test-site.com',
-  baseUrl: '/',
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
-  favicon: 'img/favicon.ico',
-  organizationName: 'facebook', // Usually your GitHub org/user name.
-  projectName: 'docusaurus', // Usually your repo name.
-  themeConfig: {
-    navbar: {
-      title: 'My Site',
-      logo: {
-        alt: 'My Site Logo',
-        src: 'img/logo.svg',
-      },
-      items: [
-        {
-          to: 'docs/intro',
-          activeBasePath: 'docs',
-          label: 'Docs',
-          position: 'left',
-        },
-        {to: 'blog', label: 'Blog', position: 'left'},
-        {
-          href: 'https://github.com/facebook/docusaurus',
-          label: 'GitHub',
-          position: 'right',
+// @ts-check
+
+module.exports =
+  // @type annotations allow IDEs to provide config autocompletion
+  /** @type {import('@docusaurus/types').Config} */
+  ({
```

Comment:
Now I think we should do

```js
/** @type {import('@docusaurus/types').Config} */
const config = {
  // ...
}
module.exports = config;
```

There's currently too much indentation, doesn't feel right for an init template

Reply:
done

we'll remove this variable later once `export default` is possible